docs/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## [Unreleased]
 
+### Changed
+
+* feat!: support for restricting II auth methods
+    * New login option: `allowPinAuthentication?: boolean;`
+    * Response from II includes `authnMethod: 'passkey' | 'pin' | 'recovery';`
+    * OnSuccess now optionally passes the message directly from the IDP provider
```

Comment:
very nice! just a thought: also allowing extra args in `login()` (sent directly to the IDP provider) would also be nice

Reply:
As in arbitrary args? I could add that